sync.sh

```diff
@@ -51,7 +51,7 @@ $working_tree_root/init-tools.sh
 
 if [ "$sync_src" == true ]; then
     echo "Fetching git database from remote repos..."
-    git fetch --all -p -v &>> $sync_log
```

Comment:
I'm not sure what the `&` was doing there to begin with, did you test if this works on Ubuntu?